tox.ini

```diff
@@ -37,48 +37,45 @@
 ; Full Examples:
 ;   - memcached-datastore_bmemcached-py37-memcached030
 ;   - python-agent_unittests-py38-with_extensions
-;   - python-adapter_gevent-py27
+;   - python-adapter_gevent-py39
 
 [tox]
 requires = virtualenv<20.22.0
```

Comment:
I think we can unpin this version of virtualenv since <v20.22.0 still support 2.7